pkg/common/constants.go

```diff
@@ -187,6 +187,7 @@ const (
 	injectSidecar                 = ".sidecar" + inject
 	InjectServerless              = "serverless" + inject             // serverless.fluid.io/inject
 	InjectUnprivilegedFuseSidecar = "unprivileged" + injectSidecar    // unprivileged.sidecar.fluid.io/inject
+	InjectVineyardSidecar         = "vineyard" + injectSidecar        // vineyard.sidecar.fluid.io/inject
```

Comment:
I think if it's no need  to set InjectVineyardSidecar by user. It should be determined automatically.

Reply:
Okay, I get it.